packages/millicast-sdk/src/workers/TransformWorker.worker.js

```diff
@@ -1,44 +1,78 @@
 import { addH26xSEI, extractH26xMetadata } from '../utils/Codecs'
 
-let uuid = ''
-let payload = ''
+const DROPPED_SOURCE_TIMEOUT = 2000
+const metadata = []
 let codec = ''
 let codecMap = {}
+const synchronizationSources = {}
+let synchronizationSourcesWithMetadata = []
 
-function createReceiverTransform () {
+function createReceiverTransform (mid) {
   return new TransformStream({
     start () {},
     flush () {},
     async transform (encodedFrame, controller) {
-      const frameCodec = codecMap[encodedFrame.getMetadata().payloadType] || codec.toUpperCase()
-      if (frameCodec === 'H264') {
-        const metadata = extractH26xMetadata(encodedFrame, frameCodec)
-        if (metadata.timecode || metadata.unregistered || metadata.seiPicTimingTimeCodeArray?.length > 0) {
-          self.postMessage({ metadata })
+      // eslint-disable-next-line no-undef
+      if (encodedFrame instanceof RTCEncodedVideoFrame) {
+        const frameCodec = codecMap[encodedFrame.getMetadata().payloadType] || codec.toUpperCase()
+        if (frameCodec === 'H264') {
```

Comment:
Will `codecMap[encodedFrame.getMetadata().payloadType]` always return uppercase codec string?

Reply:
They come from the SDP, and the cases we are testing all come in uppercase. I can add a `toUpperCase` there as well to ensure both will work